config.js

```diff
@@ -0,0 +1,10 @@
+'use strict';
+
+/* eslint no-process-env: 0 */
+module.exports = {
+  apiToken: process.env.API_TOKEN,
+  apiTokenSecret: process.env.API_TOKEN_SECRET,
+  apiBaseUrl: process.env.BASE_URL || 'https://api.smartsurvey.io/v1/surveys',
+  surveyID: process.env.SURVEY_ID || '309701',
```

Comment:
this was pointing to the test version of the survey data so it made my life easier to have it default to that, are you saying I shouldn't have a default like that? 
The plan was to default to the live one